scripts/test_adamw.py

```diff
@@ -0,0 +1,72 @@
+import torch
```

Comment:
I would remove that script, since it's not related to the current PR 😅

Reply:
oops, that one slipped through